examples/handlebars_templates/src/tests.rs

```diff
@@ -0,0 +1,82 @@
+use rocket;
+use rocket::testing::MockRequest;
+use rocket::http::Method::*;
+use rocket::http::Status;
+use rocket::Response;
+use rocket_contrib::Template;
+
+macro_rules! run_test {
+    ($req:expr, $test_fn:expr) => ({
+        let rocket = rocket::ignite()
+            .mount("/", routes![super::index, super::get])
+            .catch(errors![super::not_found]);
+
+        $test_fn($req.dispatch_with(&rocket));
+    })
+}
+
+#[test]
+fn test_root() {
+    // Check that the redirect works.
+    for method in vec![Get, Head].into_iter() {
```

Comment:
Don't use a `vec` here. You _should_ be able to do: `for method in [Get, Head].into_iter()`. You may not even need the `into_iter`.